test/blocks/app-ratings/app-ratings.test.js

```diff
@@ -0,0 +1,29 @@
+import { readFile } from '@web/test-runner-commands';
+
+const locales = { '': { ietf: 'en-US', tk: 'hah7vzn.css' } };
+window.isTestEnv = true;
+const imports = await Promise.all([import('../../../express/code/scripts/utils.js'), import('../../../express/code/scripts/scripts.js')]);
+const { getLibs } = imports[0];
+
+await import(`${getLibs()}/utils/utils.js`).then((mod) => {
+  const conf = { locales };
+  mod.setConfig(conf);
+});
+const { default: decorate } = await import('../../../express/code/blocks/app-ratings/app-ratings.js');
+const body = await readFile({ path: './mocks/body.html' });
+document.body.innerHTML = body;
+describe('App Ratings', () => {
+  before(() => {
+    window.isTestEnv = true;
+    window.placeholders = {
+      'app-store-ratings': 'Test',
+      'app-store-stars': 'Test',
+      'app-store-ratings-play-store': 'Test',
+      'app-store-ratings-apple-store': 'Test',
+    };
+  });
+
+  it('App Ratings exists', async () => {
+    await decorate(document.getElementsByClassName('app-ratings')[0]);
```

Comment:
i know sometimes some of the assertion are not really meaningful and coverage can be easily tricked, but at the very least a test file should contain at least one assertion. Maybe just expect something out of the block's html structure so that future failures can alert us about changed assumptions. Otherwise unit testing doesn't really mean anything.